index.rst

```diff
@@ -44,18 +44,589 @@
+- **When mounted on the telescope mount, with the rotator angle at zero, the x/y/z axes of the OCS are in parallel with the x/y/z axes of the OCS, and points in the same directions.**
+
+The CCS is fixed to the camera body; we use the focal plane to define the CCS because that is the only camera component that is relevant to the AOS, CS-wise. The lens surfaces do change under different gravity and thermal profile, and even the camera rotator angle. But the AOS does not actively control any camera internal components for image quality improvements.
+
+.. figure:: /_static/ccs.png
+   :name: fig-ccs
+   :target: ../_images/ccs.png
+   :alt: The CCS
+
+The Camera CS.
+
+For the wavefront sensors, the split between the intra- and extra-focal chips are parallel to the CCS y-axis on R00 and R44, and parallel to the CCS x-axis on R40 and R04. Here we refer to each 2k by 4k as one chip. Sometimes we see them refered to as half-chips as well. The one closer to the field center is always the extra-focal chip, which has larger z-coordinate in the CCS. The camera team refers to the extra-focal chip as low chip sometimes, because it is lower than the focal plane when looked through the L3 lens. For the same reason, the intra-focal chips are refered to as high chips.
+
+Two out of the four wavefront sensors (R00 and R44) have their CCD segments oriented the same way as the science sensors. Most of the science sensor segments, as seen in the CCS, has the parallel transfer direction parallel the x-axis. However, astronomers are much more used to seeing images with the parallel transfer direction going vertically, and serial register going horizontally. LSE-349 :cite:`LSE-349` defines the project's official Data Visualization CS (DVCS) as a x-y transpose of the CCS. We should be aware that most of the time when we see a visualization of certain quantities over the entire focal plane, a raft, or a single CCD, if CS is not explicitly given, the assumption should be that it is in DVCS.
+
+.. code-block:: py
+
+   # all units are mm
+   def ocs2ccs(x,y,z, d_L1_M1):
+       '''
+       d_L1_M1 is the distance between L1S1 vertex and M1 vertex.
+            it is approximately 3397mm,
+            but varies with camera hexapod positioning and filter band.
+       '''
+       return x,y,z-d_L1_M1
+   def ccs2ocs(x,y,z, d_L1_M1):
+       return x,y,z+d_L1_M1
+   def dvcs2ccs(x,y,z):
+       return y,x,z
+   def ccs2dvcs(x,y,z):
+       return y,x,z
+
+The wavefront sensors are rotated on the focal plane. The wavefront sensor images we get from the DAQ will need to be rotated to be put into the CCS. See `here in the IM code <https://github.com/bxin/IM/blob/9d74b83eb15021e91d27bf96aa262ff378550818/source/aosWFS.py#L402>`__ or `here in ts_wep code <https://github.com/lsst-ts/ts_wep/blob/master/python/lsst/ts/wep/WepController.py#L408-L421>`__.
```

Comment:
There was the discussion to unify the coordinate systems of corner WFS. If this was the case in the final, we would not need to do this rotation anymore. The decision and related work are still TBD at this moment.

Reply:
will add a note. thanks.